backend/repositories/ContentRepository.ts

```diff
@@ -23,13 +41,12 @@ export type ContentSearchParametersType = {
   blockFrom?: number;
   blockTo?: number;
   contentHash?: string;
+  relatedContentHash?: string;
```

Comment:
Thinking about naming here. `inReplyToContentHash`, `parentContentHash`. @mattheworris @JoeCap08055 thoughts?

Reply:
I chose `relatedContentHash` because it's not just for replies, it's also relevant for Reactions, Updates, and Tombstones.